tedana/workflows/tedana.py

```diff
@@ -422,15 +430,21 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     if png:
         LGR.info('Making figures folder with static component maps and '
                  'timecourse plots.')
+        # make figure folder first
+        if not os.path.isdir('figures'):
```

Comment:
I think I did a bad job using the suggestion function in GitHub. You need to delete this line now.

Reply:
gotcha